x-pack/plugins/observability/public/components/app/observability_status/observability_status_box.tsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiBadge,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPanel,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+interface Props {
+  dataSource: string;
+  hasData: boolean;
+  description: string;
+  modules: Array<{ name: string; hasData: boolean }>;
+  integrationLink: string;
+  learnMoreLink: string;
+}
+
+export function ObservabilityStatusBox(props: Props) {
+  if (props.hasData) {
+    return <CompletedStatusBox {...props} />;
+  } else {
+    return <EmptyStatusBox {...props} />;
+  }
+}
+
+export function CompletedStatusBox({ dataSource, modules, integrationLink }: Props) {
+  return (
+    <EuiPanel color="subdued" style={{ marginBottom: 20 }}>
+      <EuiFlexGroup justifyContent="spaceBetween">
+        <EuiFlexItem>
+          <div>
+            <EuiIcon
+              type="checkInCircleFilled"
+              color="success"
+              className="eui-displayInline eui-alignMiddle"
```

Comment:
[nit] I'm not sure how common is to use classes within Kibana. Maybe it would make sense to wrap the `EuiIcon` and the `EuiTitle` individually in a `EuiFlexItem` ?

Reply:
These classes are documented in the [EUI library ](https://elastic.github.io/eui/#/utilities/css-utility-classes) so I assumed that we can use them. I don't like to add components just to make small style fixes that can be done with CSS, but if that's not recommended from Kibana I can change it.

Reply:
To be honest I don't know, but if it's part of EUI let's keep it